packages/ihe-gateway/server/Channels/CareQuality Document Exchange - Inbound/XCA ITI-38 Inbound Interface/destinationConnector-AWS Lambda Interface-responseTransformer-step-0-Process response.js

```diff
@@ -1,4 +1,38 @@
 // Store the HTTP code only
+
+// old post processing script from processor
```

Comment:
Why not moved to post processor of the channel?

Reply:
And AFAIK this should be the very last thing to be executed on the channel, no?

Reply:
Ditto re: was postprocessing for when there were 2 channels but now there is only one